appengine/py_appengine.bzl

```diff
@@ -199,8 +199,9 @@ def py_appengine_binary(name, srcs, configs, deps=[], data=[]):
   # uses py_binary because it generates __init__.py files
   native.py_binary(
       name="_py_appengine_" + name,
-      srcs = srcs,
+      srcs=srcs,
```

Comment:
This is not a cleanup.  Skylark style has spaces around '=' which is different than python